src/scrapers/games_scraper.py

```diff
@@ -8,6 +8,73 @@
 from src.utils.helpers import get_dominant_color
 import base64
 import html
+import pytz
+
+def parse_time_string(time_str):
+    """
+    Parse time into a standardized format (e.g. 12:01 AM)
+    """
+    if not time_str:
+        return None
+        
+    time_str = time_str.strip().lower()
+    
+    # sometimes written as p.m.
+    time_str = time_str.replace("p.m.", "pm").replace("a.m.", "am")
+    
+    # extract hours, minutes, and AM/PM
+    time_pattern = re.compile(r'(\d{1,2}):?(\d{2})?\s*(am|pm)?')
+    match = time_pattern.search(time_str)
+    
+    if not match:
+        return None
+        
+    hours = match.group(1)
+    minutes = match.group(2) or "00"
+    
+    # default to pm if not specified
+    am_pm = match.group(3) or "pm" 
+    
+    return f"{hours}:{minutes} {am_pm.upper()}"
+
+def convert_to_utc(date_str, time_str, eastern_tz=pytz.timezone('US/Eastern')):
```

Comment:
this function can probably be extracted into a utility file that we can call everywhere in the project